pytest_fixtures/api_fixtures.py

```diff
@@ -560,23 +553,19 @@ def module_env():
 
 
 @pytest.fixture(scope='module')
-def module_env_search(module_org, module_location, module_cv_with_puppet_module):
-    """Search for puppet environment according to the following criteria:
-    Content view from module_cv_with_puppet_module and chosen organization.
+def module_env_search(module_org, module_location, module_import_puppet_module):
+    """Search for puppet environment created from module_import_puppet_module fixture.
 
-    Returns the puppet environment with updated location.
+    Returns the puppet environment with updated organization and location.
     """
     env = (
         entities.Environment()
-        .search(
-            query={
-                'search': f'content_view={module_cv_with_puppet_module.name} '
-                f'and organization_id={module_org.id}'
-            }
-        )[0]
+        .search(query={'search': f'name={module_import_puppet_module}'})[0]
         .read()
     )
-    env.location.append(module_location)
+    env.organization = [module_org]
```

Comment:
Why is this done as separate calls instead of updating both fields at once?

Reply:
I don't know :smile: :smile: So I will change it too.